ex3/index.js

```diff
@@ -0,0 +1,15 @@
+function letterCount(sir, litera) {
+    let sirLower = sir.toLowerCase();
+    let literaLower = litera.toLowerCase();
+    let count = 0;
+    for (let i = 0; i < sirLower.length; i++) {
+    if (sirLower[i] === literaLower) {
+    count++;
+    }
+    }
+   
+    return count;
+   }
+   
+   console.log(letterCount("Îmi place programarea", "a"));
+   console.log(letterCount("Vreau sa lucrez în IT", "r"));
```

Comment:
verifica te rog de cate ori apare litera "t' aici? in Loc de "r".
Daca apare de 0 ori, inseamna ca functia ta tine cont de majuscula/minuscula si se poate de fixat